Tests/Unit/Form/Type/StaticContentTypeTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Tests\Unit\Form\Type;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+use Ivory\CKEditorBundle\Model\ConfigManagerInterface;
+use Ivory\CKEditorBundle\Model\PluginManagerInterface;
+use Ivory\CKEditorBundle\Model\StylesSetManagerInterface;
+use Ivory\CKEditorBundle\Model\TemplateManagerInterface;
+use Symfony\Cmf\Bundle\ContentBundle\Form\Type\StaticContentType;
+use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\PreloadedExtension;
+use Symfony\Component\Form\Test\TypeTestCase;
+
+class StaticContentTypeTest extends TypeTestCase
+{
+    private $configManager;
+    private $pluginManager;
+    private $stylesSetManager;
+    private $templateManager;
+
+    protected function setUp()
+    {
+        $this->configManager = $this->getMock(ConfigManagerInterface::class);
+        $this->pluginManager = $this->getMock(PluginManagerInterface::class);
+        $this->stylesSetManager = $this->getMock(StylesSetManagerInterface::class);
+        $this->templateManager = $this->getMock(TemplateManagerInterface::class);
+        parent::setUp();
+    }
+
+    protected function getExtensions()
+    {
+        $type = new CKEditorType(
+            $this->configManager,
+            $this->pluginManager,
+            $this->stylesSetManager,
+            $this->templateManager
+        );
+
+        return [
+            new PreloadedExtension([$type], []),
+        ];
+    }
+
+    public function testSubmitValidData()
+    {
+        $data = [
+            'title' => 'Hello World!',
+            'body' => '<p>Lorem ipsum dolor sit amet.</p>',
+        ];
+
+        $form = $this->factory->create(StaticContentType::class);
+
+        $form->submit($data);
+
+        $object = new StaticContent();
+        $object->setTitle($data['title']);
```

Comment:
Does starting with an empty `$object` is better? Anyway `$form->submit($data)` will fulfill `title` and `body`!